### PR TITLE
feat: add schema for timely

### PR DESCRIPTION
**Pull request by moripen**

### CODE REVIEW

**Review by simenandre — COMMENTED** [review]
Can we add a name, and use that alone in the preview instead of a bunch of values?

File: schemas/timely.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      title: 'Employee Token',
+      description: `
+        The employee token to use when creating new Tripletex time entries.
+      `
+    },
+    {
+      name: 'employees',
+      type: 'array',
+      title: 'Employees',
+      of: [{ type: 'string', title: 'email' }],
```
Comment (simenandre) [inline schemas/timely.ts]: I think this might miss a value?

Comment (moripen) [reply schemas/timely.ts]: Does it? What value is missing?

File: schemas/objects/employee.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { defineType } from 'sanity';
+
+export default defineType({
+  name: 'employee',
```
Comment (simenandre) [inline schemas/objects/employee.ts]: Should we name this something so it's clear this only is for the Timely integration?

Comment (moripen) [reply schemas/objects/employee.ts]: something like employeeTimely?

File: schemas/timely.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
+export default {
```
Comment (simenandre) [inline schemas/timely.ts]:
```suggestion
import { defineType } from 'sanity';

export default defineType({
```

**Review by simenandre — APPROVED** [review]

**Pull request merged**